ossdbtoolsservice/driver/types/pymysql_driver.py

```diff
@@ -63,6 +65,11 @@ def __init__(self, conn_params):
                 if val:
                     self._connection_options[param] = int(val) or None
 
+        # Use the default database if one was not provided
+        if 'database' not in self._connection_options or not self._connection_options['database']:
+            if configs:
+                self._connection_options['database'] = configs.my_sql.default_database
```

Comment:
Why is this my_sql, should it not be configs.mysql

Reply:
It's how sasank defined the variable in MySQLConfiguration. Would you like me to change it to 'mysql'?

Reply:
No lets not do it right now. We can do it separately.